plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/BaseMySqlConnectorTest.java

```diff
@@ -337,6 +342,87 @@ public void testPredicatePushdown()
                 .isFullyPushedDown();
     }
 
+    @Test
+    public void testPredicatePushdownWithCollation()
+    {
+        onRemoteDatabase().execute("CREATE TABLE tpch.nation_collate AS SELECT regionkey, nationkey, CONVERT(name USING utf8) COLLATE utf8_bin AS name FROM tpch.nation");
```

Comment:
can you have multiple tests to test different collations?

Reply:
added